src/scripts.js

```diff
@@ -37,12 +53,44 @@ quoteBtn.addEventListener('click', displayTripQuote);
 clearFormBtn.addEventListener('click', clearForm);
 newTripForm.addEventListener('submit', packageNewTrip);
 
+loginForm.addEventListener('submit', validateLogin);
+
+
 // Main Functions -------------------------------------------------------------------------------------------
 
-function displayDashboard(e) {
-  createDashboardView(33, e);
+function validateLogin(e) {
+  e.preventDefault();
+  const username = usernameInput.value;
+  const password = passwordInput.value;
+  const splitUsername = username.split('');
+  const usernameLetters = splitUsername.slice(0, 8).join('');
+  const usernameNumbers = splitUsername.slice(8, 11).join('');
+
+  domUpdates.addTravelerIDToForm(parseInt(usernameNumbers));
+
+  domUpdates.validateUsername(usernameLetters, usernameNumbers)
+  domUpdates.validatePassword(password)
+
+  if (usernameInput.classList.contains('correct') && passwordInput.classList.contains('correct')) {
+    usernameError.innerText = "";
+    passwordError.innerText = "";
+    loadDashboardAfterLogin(e)
+  } 
+}
```

Comment:
This function feels a tad long? I would consider chaining to reduce variable naming, or keeping the `if` block in this function and moving the variable naming and DOM stuff into a separate function that gets invoked here.